pkg/scheme/core/v1/cri/containerd.go

```diff
@@ -256,3 +261,166 @@ func (runnable *ContainerdRunnable) renderTo(w io.Writer) error {
 	_, err := at.RenderTo(w, configTomlTemplate, runnable)
 	return err
 }
+
+func (runnable *ContainerdRunnable) renderRegistryConfig(dryRun bool) error {
+	if dryRun {
+		return nil
+	}
+	configDir := containerdDefaultRegistryConfigDir
+	if runnable.RegistryConfigDir != "" {
```

Comment:
where is  RegistryConfigDir from? Does front add a field？

Reply:
This field is currently reserved to enhance customizability, and can be obtained from the web page later